client/my-sites/site-settings/form-writing.jsx

```diff
@@ -195,6 +201,9 @@ const connectComponent = connect(
 		const siteIsJetpack = isJetpackSite( state, siteId );
 		const siteIsAutomatedTransfer = isSiteAutomatedTransfer( state, siteId );
 		const isPodcastingSupported = ! siteIsJetpack || siteIsAutomatedTransfer;
+		const isNavUnification = isNavUnificationEnabled( state );
```

Comment:
Not sure if checking for isNavUnificationEnabled is necessary but I figured it couldn't hurt...

Reply:
What happens if we don't use it? If it's not needed, it's good not to introduce more parameters.